src/main/protobuf/com/rawlabs/protocol/das/v1/common/environment.proto

```diff
@@ -18,5 +18,6 @@ option java_multiple_files = true;
 package com.rawlabs.protocol.das.v1.common;
 
 message Environment {
-  map<string, string> env = 2;
-}
\ No newline at end of file
+  repeated string scopes = 1;
+  map<string, string> http_headers = 2;
```

Comment:
By renaming the `env` field, we ensure backward compatibility between two systems where one would be using the new version, and one would be using the old version.

Now if an old client pulls the new `.proto` files and regenerates the API (Python, Scala, etc.), its code isn't going to work anymore.